receiver/otlpreceiver/testdata/config.yaml

```diff
@@ -2,51 +2,72 @@ receivers:
   # The following entry initializes the default OTLP receiver.
   # The full name of this receiver is `otlp` and can be referenced in pipelines by 'otlp'.
   otlp:
+    protocols:
+      grpc:
+      http:
```

Comment:
These are required to enable the protocol, right? If any of these lines is missing that particular protocol will not be enabled. Do I understand it correctly?

Reply:
Commented in the README about this.